osdconfig/api.go

```diff
@@ -76,7 +49,7 @@ func (manager *configManager) getStatus() map[string]*Status {
 	return M
 }
 
-// Error returns error status of callback execution
+// error returns error status of callback execution
 func (manager *configManager) error() error {
```

Comment:
Please change the name of this function. error() looks like `error` the type.

Reply:
will do